tools/benchmark/src/MochaReporter.ts

```diff
@@ -85,6 +85,15 @@ module.exports = class {
 					benchmarkReporter.recordSuiteResults(getSuiteName(suite));
 				}
 			})
+			.on(Runner.constants.EVENT_HOOK_END, (hook: Hook) => {
```

Comment:
Related, I was experimenting with how to get errors in hooks to cause the test to get reported as failed but could not get to the behavior I was looking for. Not entirely sure it's possible. Logging to console if it happens seems like a good improvement, even if it requires FLUID_TEST_VERBOSE=1 to be set.

Reply:
note that this after each error actually showed up as a test failed event, not hook failed, but I kept this code incase it helps a different case.